src/templates/doe_workflow_attic_air_sealing.mdx

```diff
@@ -0,0 +1,291 @@
+<Tabs>
+<Tab eventKey="project" title="Project">
+    The data for this tab should be known prior to the initial site visit.
+
+    <ProjectInfoInputs {...props} />    
+  </Tab>
+  <Tab eventKey="assessment" title="Assessment">
+    The data for this tab should be collected during the initial site visit.
+     
+    ## Building 
+    <PhotoInput id="building_number_photo" label="Building Number – Photo">
+      Provide a photo of the building that shows the street number.
+    </PhotoInput>
+
+    ## Blower Door
+    <PhotoInput id="blower_door_setup_photo" label="Blower Door Setup – Photo">
+        Photo of the blower door setup before the air sealing insulation work
+    </PhotoInput>
+    <PhotoInput id="cfm50_photo" label="CFM50 – Photo">
+        Photo of the manometer showing CFM50 of air leakage before air sealing and insulation work was performed
+    </PhotoInput>
+    | ACH50 Calculator                                                                                                                                                                                                                                           |
+    | ----------------------------
+    | <NumberInput label="CFM50" min={0} path="ACH50_calculator.cfm50"  /> 
```

Comment:
For consistency, let's only use lower-case in paths, so "ach50_calculator.cfm50" here. Similarly below.

Reply:
changed to lowercase in both the new templates